mor/pyfeelpp-mor/feelpp/mor/pbdw.cpp

```diff
@@ -0,0 +1,59 @@
+//! -*- mode: c++; coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; show-trailing-whitespace: t  -*- vim:fenc=utf-8:ft=cpp:et:sw=4:ts=4:sts=4
+//!
+//! This file is part of the Feel++ library
+//!
+//! This library is free software; you can redistribute it and/or
+//! modify it under the terms of the GNU Lesser General Public
+//! License as published by the Free Software Foundation; either
+//! version 2.1 of the License, or (at your option) any later version.
+//!
+//! This library is distributed in the hope that it will be useful,
+//! but WITHOUT ANY WARRANTY; without even the implied warranty of
+//! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//! Lesser General Public License for more details.
+//!
+//! You should have received a copy of the GNU Lesser General Public
+//! License along with this library; if not, write to the Free Software
+//! Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+//!
+//!
+#include <pybind11/pybind11.h>
+#include <pybind11/eigen.h>
+#include <pybind11/stl.h>
+
+#include <feel/feelmor/pbdw.hpp>
+
+namespace py = pybind11;
+
+PYBIND11_MODULE(_pbdw, m )
+{
+    using namespace Feel;
+    using pbdw_t = PBDWOnline;
+    using vectorN_type = typename pbdw_t::vectorN_type;
+    using matrixN_type = typename pbdw_t::matrixN_type;
+    py::class_<pbdw_t,std::shared_ptr<pbdw_t>>(m, "Pbdw")
+        .def(py::init<std::string const&, int, std::string const&, std::string const&>(),
+             py::arg("name")=std::string("pbdw"),
+             py::arg("dbLoad")=1,
```

Comment:
should be an enum no ?

Reply:
this is the general behavior of the option in crb, I didn't want to change everything, so I used the same thing.